src/main/java/apoc/text/Strings.java

```diff
@@ -84,6 +86,27 @@ public boolean compareCleaned(final @Name("text1") String text1, final @Name("te
     }
 
     @UserFunction
+    @Description("apoc.text.getLevenshteinDistance(text1, text2) - compare the given strings with the StringUtils.getLevenshteinDistance(text1, text2) method")
+    public Long getLevenshteinDistance(final @Name("text1") String text1, @Name("text1")final String text2) {
```

Comment:
how does it handle it if one or both are null?
perhaps return null then like the rest of cypher